proxy/http/HttpTransact.h

```diff
@@ -923,6 +931,10 @@ class HttpTransact
 
     NetVConnection::ProxyProtocol pp_info;
 
+  private:
+    // Make this a raw byte array, so it will be accessed through the my_txn_conf() member function.
+    alignas(OverridableHttpConfigParams) char _my_txn_conf[sizeof(OverridableHttpConfigParams)];
```

Comment:
Just as an encouragement when new overridable configs are added, to use the the my_txn_conf() member function rather than directly changing the _my_txn_conf.  The member function has an ink_assert to verify you're not changing the per-txt copy without first copying in the global baseline values.